adding S3 bucket directory. (#5)
invenia/OPFSampler.jl: README.md

# Goal
The goal of the package is to provide functions that take a power grid as input, vary its parameters and generate feasible DC- and AC-OPF samples along with the corresponding solutions. This helps the user to explore a variety of distinct active sets of constraints of synthetic cases and mimic the time-varying behaviour of the OPF input parameters.

> We have made a publicly-available database of input samples for different grids for both DC- and AC-OPF that have feasible OPF solution, which can be found in the following Amazon S3 bucket (s3://invenia-public-datasets/OPFSamples/pglib-opf/). There are currently two folders in there corresponding to 10k DC-OPF samples and 1k AC-OPF samples for different grids.  

Reviewer: ```suggestion
We created a publicly-available database of input samples for different grids for both DC- and AC-OPF that have feasible OPF solution. The samples are collected in the following Amazon S3 bucket: `s3://invenia-public-datasets/OPFSamples/pglib-opf/`. There are currently two folders in the main directory corresponding to 10k DC-OPF and 1k AC-OPF samples for different grids.  
```

| 4661-sdet    |       10k      |        -       |
| 9241-pegase  |       10k      |        -       |

In the generated samples above, the OPF solution that is generated under "OPF_output" has been removed to keep the size of the data small.
## How to Download Generated Samples?
The generated samples are hosted in Amazon S3 bucket (s3://invenia-public-datasets/OPFSamples/pglib-opf/). The following folders are available in this directory:

> * `input_10k_DC_0.15pd_0.1rest`: This folder contains 10k DC-OPF samples for the grids given in the above table. As indicated by the folder name, "dev_load_pd" is chosen to be 0.15 while other deviation parameters in `params_DC` are set to 0.1.

Reviewer: ```suggestion
* `input_10k_DC_0.15pd_0.1rest`: This folder contains 10k DC-OPF samples for the grids given in the table above. As indicated by the folder name, "dev_load_pd" is chosen to be 0.15 while other deviation parameters in `params_DC` are set to 0.1.
```

> * `input_1k_AC_0.15pd_0.1rest`: This folder contains 1k AC-OPF samples for the grids given in the above table. As indicated by the folder name, "dev_load_pd" is chosen to be 0.15 while other deviation parameters in `params_AC` are set to 0.1.

Reviewer: ```suggestion
* `input_1k_AC_0.15pd_0.1rest`: This folder contains 1k AC-OPF samples for the grids given in the table above. As indicated by the folder name, "dev_load_pd" is chosen to be 0.15 while other deviation parameters in `params_AC` are set to 0.1.
```
